### ISSUE TITLE
paper-input values get submitted twice (-> as array)

### ISSUE THREAD

**Issue by flaiker** [rnicholus/ajax-form#78]
Hi,

I noticed when I use paper-input elements (which are supported according to the readme) every value gets submitted twice on submit, making it an array when receiving it.

Heres the minimal code to reproduce:
```html
<!DOCTYPE html>
<html>
<head>
    <script src="bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
    <link rel="import" href="bower_components/paper-input/paper-input.html">
    <link rel="import" href="bower_components/ajax-form/ajax-form.html">
</head>
<body>
<form is="ajax-form" method="post" enctype="application/json">
    <paper-input name="username" value="testname" type="text"></paper-input>
    <input type="submit">
</form>
</body>
</html>
```

And the result:
```json
{"username":[["testname"],["testname"]]}
```

Seems to be independent of method and encodingtype.

I assume it is because paper-input wraps a normal input element in itself and so they both are evaluated by the value-retrieval function.

**Comment by rnicholus** [comment]
The issue you describe sounds identical to #73. I created a hotfix for the issue at https://github.com/rnicholus/ajax-form/tree/hotfix/2.1.1, but the person who opened #73 never responded, so I closed out the issue and abandoned the hotfix branch. Please take a look at the branch at let me know if it fixes your issue. If it does, I'll be happy to release the fix.

**Comment by flaiker** [comment]
Yup. sorry for not looking at closed issues beforehand.
The hotfix/2.1.1 branch fixes it for me.

**Comment by rnicholus** [comment]
Thanks for the confirmation. I'll release #73 as 2.1.3 then.

Duplicate of #73.

**Comment by rnicholus** [comment]
I just released 2.1.3 with this fix. Also published to npm.

**Comment by flaiker** [comment]
Thanks for your fast fix!
